pandas/tests/indexing/test_loc.py

```diff
@@ -1955,6 +1955,13 @@ def test_loc_setitem_dt64tz_values(self):
         result = s2["a"]
         assert result == expected
 
+    def test_loc_setitem_boolean_list(self):
```

Comment:
lets put this in TestLocBooleanMask

can this be shared by DataFrame?

few words after the GH ref to the effect of "specifically list key, not arraylike"

Reply:
Done, added a ``DataFrame`` part to the test, parametrization would be quite ugly here